### PR TITLE
Add small image Auto-generated

### PR DESCRIPTION
**Pull request by joerkig**
Adds a small image to indicate that user is showing auto-generated beatmaps.

Show auto-generated beatmaps checked:
Discord_Qpz689BNUK
Discord_8sUKSdsNTN
Discord_wH0DMScJAI

Show auto-generated beatmaps unchecked:
Discord_f9KkpBpRwv
Discord_yTf8VxYmdN
Discord_1mkLzsOikt

Works on every Browse page and the Search and Uploader pages

### CODE REVIEW

File: websites/B/BeatSaver/presence.ts [hunk 1]
```diff
@@ -51,17 +56,23 @@ presence.on("UpdateData", async () => {
...
+    presenceData.state = document
+      .querySelector("h1.is-size-2.has-text-weight-light.has-text-centered")
+      .textContent.split(" ")[2];
+    presenceData.buttons = [
+      {
+        label: "View Page",
+        url: document.location.href
+      }
+    ];
+    if (showAutos.checked == true) {
```
Comment (ririxidev) [inline websites/B/BeatSaver/presence.ts:70]:
```suggestion
    if (showAutos.checked) {
```

File: websites/B/BeatSaver/presence.ts [hunk 2]
```diff
@@ -10,13 +10,18 @@ presence.on("UpdateData", async () => {
...
-    };
+    },
+    showAutos = <HTMLInputElement>document.querySelector("#showAutos");
 
   if (document.location.pathname.includes("/search")) {
     presenceData.details = "Searching Beatmaps";
     presenceData.state = document
       .querySelector("input.input")
       .getAttribute("value");
+    if (showAutos.checked == true) {
```
Comment (ririxidev) [inline websites/B/BeatSaver/presence.ts:21]:
```suggestion
    if (showAutos.checked) {
```

File: websites/B/BeatSaver/presence.ts [hunk 3]
```diff
@@ -101,6 +112,13 @@ presence.on("UpdateData", async () => {
       break;
   }
 
+  if (document.location.pathname.split("/")[1].includes("browse")) {
+    if (showAutos.checked == true) {
```
Comment (ririxidev) [inline websites/B/BeatSaver/presence.ts:116]:
```suggestion
    if (showAutos.checked) {
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

**Comment by ririxidev** [comment]
Well, resolve conflicts btw.

**Comment by ririxidev** [comment]
and now make changes to metadata since @SeMiD was prettifying it

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**